### PR TITLE
Add format check on MediaSaveManager

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/86
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/135

### CODE REVIEW

File: Media/Manager/SaveMediaManager.php [hunk 1]
```diff
@@ -15,20 +15,24 @@ class SaveMediaManager implements SaveMediaManagerInterface
...
      * @param UploadedMediaManager $uploadedMediaManager
+     * @param array                $allowedMimeTypes
      */
     public function __construct(
         $tmpDir,
         ThumbnailManager $thumbnailManager,
-        UploadedMediaManager $uploadedMediaManager
-    ) {
+        UploadedMediaManager $uploadedMediaManager,
+        $allowedMimeTypes
```
Comment (nicolasThal) [inline Media/Manager/SaveMediaManager.php]: array $allowedMimeTypes

File: MediaBundle/Tests/Manager/SaveMediaManagerTest.php [hunk 2]
```diff
@@ -105,6 +109,27 @@ public function provideMedias()
     }
 
     /**
+     * @param string $filename
+     * @param bool   $expectedStatus
+     *
+     * @dataProvider provideFileType
+     */
+    public function testIsFileAllowed($filename, $expectedStatus) {
```
Comment (nicolasThal) [inline MediaBundle/Tests/Manager/SaveMediaManagerTest.php]: psr for the `{`

**Comment by itkg-canne** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
@nicolasThal : update done

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**